api/integrations/github/views.py

```diff
@@ -71,6 +99,22 @@ def create(self, request, *args, **kwargs):
             if re.search(r"Key \(organisation_id\)=\(\d+\) already exists", str(e)):
                 raise DuplicateGitHubIntegration
 
+    def destroy(self, request, *args, **kwargs):
+        github_response = delete_github_installation(self.get_object().installation_id)
+        if github_response.status_code != 204:
+            return Response(
```

Comment:
Why no decorator here? Related to [this comment](https://github.com/Flagsmith/flagsmith/pull/3962/files#r1605261746).

Reply:
Being a class View it is no decorator is needed here.

Reply:
You should be able to decorate class methods without any problems.

Reply:
Yes, but I don't see that we need a decorator, here. Maybe I misunderstood your comment, Which decorator do you think we are missing here?

Reply:
The one you wrote to wrap GitHub errors, `github_api_call_error_handler`.

Reply:
So it was just easier this way

Reply:
You are using a generic `*args, **kwargs` signature when wrapping, so it shouldn't be a problem, unless I'm missing something here.

Reply:
Oh you're right, I was thinking all the time in this other decorator I wrote which has a specific signature https://github.com/Flagsmith/flagsmith/blob/feat/uninstall-github-app-installation-from-flagsmith/api/integrations/github/views.py#L33-L36
I'll see if can apply it to this class View.